tach.toml

```diff
@@ -0,0 +1,39 @@
+exact = true
+exclude = []
+forbid_circular_dependencies = true
+source_roots = [
+  "src"
+]
+
+[[modules]]
+depends_on = [
+  {path = "gt4py.eve"}
+]
+path = "gt4py._core"
```

Comment:
I guess the order is coming from the formatter?

Reply:
Yes, exactly, but in this case I don't think it makes sense so I'll exclude this file from the toml autoformatter.